Runtime/Behaviours/CinemachineBlendListCamera.cs

```diff
@@ -278,12 +278,14 @@ private void OnGuiHandler()
                 CinemachineDebug.ReleaseScreenPos(this);
             else
             {
+#if CINEMACHINE_UNITY_IMGUI
```

Comment:
Need at the top:
#if !UNITY_2019_3_OR_NEWER
#define CINEMACHINE_UNITY_IMGUI
#endif